deployment/tfhub/object-detection/base64/object-detection-base64-predict.ipynb

```diff
@@ -1,241 +1,293 @@
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Object Identification (TensorFlow Hub) - Base64 Images"
-   ]
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "99FgJuGtHbK-"
+      },
+      "source": [
+        "# Object Detection - SSD Mobilenet V2 from TensorFlow Hub (Base64 Images)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "aYVICIPxIqAM"
+      },
+      "source": [
+        "This model identifies objects present in images, returning its scores, labels and bounding boxes.\n",
+        "\n",
+        "We are using a dataset from [UCF](https://www.crcv.ucf.edu/data/GMCP_Geolocalization/#Dataset) and the model [SSD Mobilenet V2](https://tfhub.dev/google/openimages_v4/ssd/mobilenet_v2/1) from [TensorFlow Hub](https://www.tensorflow.org/hub)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Wh9NwPMRIu5n"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/VertaAI/examples/blob/main/deployment/tfhub/object-detection/base64/object-detection-base64-predict.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Fm0ffqV_Iuw_"
+      },
+      "source": [
+        "## Dependencies"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qJSXlocsIuln"
+      },
+      "source": [
+        "This notebook has been tested with **Python 3.8.15** and the following package versions:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "YVkBoQBGzdd-"
+      },
+      "outputs": [],
+      "source": [
+        "%%capture\n",
+        "!pip install numpy==1.23.5\n",
+        "!pip install verta==0.21.1"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8aNS5l4PJ__e"
+      },
+      "source": [
+        "## Imports"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ASXSPO_ezkT-"
+      },
+      "outputs": [],
+      "source": [
+        "import base64\n",
+        "import concurrent.futures\n",
+        "import numpy as np\n",
+        "import os\n",
+        "import requests\n",
+        "import time\n",
+        "import json\n",
+        "\n",
+        "from verta import Client"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p-JtMXyhKK83"
+      },
+      "source": [
+        "## Verta Set Up"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "cXeEZ9yRKKXf"
+      },
+      "outputs": [],
+      "source": [
+        "os.environ['VERTA_HOST'] = ''\n",
```

Comment:
The `deploy` notebook has

```
VERTA_HOST = 'app.verta.ai'
```

so it might make sense to include it in the `predict` notebook also.

```suggestion
        "os.environ['VERTA_HOST'] = 'app.verta.ai'\n",
```

Reply:
I removed it from `deploy`  as well, to follow the same pattern of other notebooks (Verta's credentials always empty).